Automattic/woocommerce-payments
# Set description when creating payment intents

Fixes #1807

#### Changes proposed in this Pull Request

Adds a description when creating payment intents. This description contains the order ID (if available), the store's domain name, and the blog ID (if available).

Follows a similar approach to https://github.com/woocommerce/woocommerce-ios/pull/4373

![124856684-51a65d80-dfee-11eb-91aa-26114d0b8a01](https://user-images.githubusercontent.com/1527164/124866362-a6060900-dfff-11eb-9db4-aef7a0039350.png)

#### Testing instructions

* Add items to the cart, complete checkout.
* Log into the Stripe dashboard and navigate to your connected account.
* Navigate to the list of payments.
* Observe a description is visible on the most recent transaction, e.g.
> Online Payment for Order #[Order ID] for [domain name] blog_id [blog ID]

-------------------
